### PR TITLE
DB-860 : Add TokuDB-InnoDB replication tests

### PR DESCRIPTION
**Pull request by george-lorch**
- Implemented basic TokuDB to InnoDB and InnoDB to TokuDB replication tests for type compatibiity.

### CODE REVIEW

**Comment by georgelorchpercona** [comment]
jenkins : http://jenkins.percona.com/view/PS%205.6/job/percona-server-5.6-param/998/

**Comment by george-lorch** [comment]
jenkins http://jenkins.percona.com/view/PS%205.6/job/percona-server-5.6-param/999/

File: mysql-test/suite/tokudb.rpl/include/rpl_tokudb_innodb.inc [hunk 1]
```diff
@@ -0,0 +1,213 @@
+# Tests basic replication interoperability between tokudb and other storage
+# engines, specifically innodb.
+# Credit to Sveta Smirnova for the basic idea.
+connection slave;
+stop slave;
+start slave;
+connection master;
+
+-- disable_query_log
+set sql_log_bin=0;
```
Comment (laurynas-biveinis) [inline mysql-test/suite/tokudb.rpl/include/rpl_tokudb_innodb.inc:10]: I don't see this setting being restored after DDL?

Comment (george-lorch) [inline mysql-test/suite/tokudb.rpl/include/rpl_tokudb_innodb.inc:10]: Great catch, I accidentally deleted the re-enabling of the binlog when refactoring the original case into a generic case to test both directions...killing this PR and will resubmit once fixed and retested.

**Pull request closed without merging**